src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php

```diff
@@ -511,6 +511,12 @@ private function getArgumentsAsPhp(\DOMElement $node, $name, $file, $lowercase =
                     }
                     $arguments[$key] = new TaggedIteratorArgument($arg->getAttribute('tag'));
                     break;
+                case 'binary':
```

Comment:
It this really legitimate? I mean: if `!!binary` is a core yaml feature, it should be transparent. Yaml tagged-values are only for custom tags, not core ones AFAIK.

Reply:
I looked for a way to indicate in a XML file (which should be a valid UTF-8 file) that a value was `base64_encoded()`. I though this was the exact same purpose as in yaml file. I did not knew that `!!binary` was part of the yaml specification.

Reply:
IMO using the same term here is legitimate. I would keep it as is.

Reply:
yes, sorry, I misread the code, all good here